Surface_mesh/benchmark/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ include_directories(BEFORE "../include")
 #add_definitions("-pg")
 #SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pg")
 # add_definitions("-g")
-add_definitions("-std=c++11")
+add_definitions("-std=c++1y")
```

Comment:
I think none of that is relevant. CMake handles the dependency on C++14 here:
https://github.com/CGAL/cgal/blob/328f2bdbc6f9d2056e967b8d5fbc5ce025dbe361/Installation/cmake/modules/CGAL_SetupCGALDependencies.cmake#L104-L106

Reply:
Well, it seemed to be causing errors in travis, with this C++11 flag overriding the C++14 one from cmake, and it seemed easier for someone who doesn't know this configuration to replace 11 with 1y than remove the line completely, but if you want to remove it, please do.

Reply:
You are right that:
  - our CMake scripts add `-std=gnu++14` (or similar),
  - but maybe the flags specified on the command line, or with `add_definitions` are after.